modules/installation-aws-limits.adoc

```diff
@@ -71,7 +70,7 @@ To use the `us-east-1` region, you must increase the EIP limit for your account.
 |20 per region
 |By default, each cluster creates internal and external network load balancers for the master
 API server and a single classic elastic load balancer for the router. Deploying
-more Kubernetes LoadBalancer Service objects will create additional
+more Kubernetes `LoadBalancer` service objects will create additional
```

Comment:
There is no LB service object.

There is a k8s `Service` object with `type=LoadBalancer` however, and that's what this is referring to.